docker-compose.yaml

```diff
@@ -79,15 +100,15 @@ services:
     image: content-watcher-service
     build:
       context: .
-      dockerfile: dev.Dockerfile
+      dockerfile: Dockerfile
```

Comment:
nit: just that dev.Dockerfile would have rebuilt project on changes to code files, but i guess a tradeoff between making it work for mac/linux

Reply:
Yeah, I don't know if I want to commit this change, but it did sort of hide the fact that the container wasn't working, so we might need 2 versions, one for development and one for testing the production image. 🤔

Reply:
dev.Dockerfile works fine for me. The point is, it's only for local development. Assuming your container & local environment are the same architecture, you have to manually manage your node_modules folder outside of the container.